### ISSUE TITLE
Cannot find module './obs_studio_client.node' or The file cannot be accessed by the system.

### ISSUE THREAD

**Issue by Tanoshi0625** [streamlabs/obs-studio-node#1426]
My environment：
OS：Windows 10 22H2
Node: v18.19.0
OSN: 0.23.107
YARN：1.22.21

Please help me, How to use osn in win64?

When i using "require", I got this error.
1. Error: The file cannot be accessed by the system.

When i using "import", I got this error. 
2. Error: Cannot find module './obs_studio_client.node'

My demo：
[https://github.com/Tanoshi0625/test-program](url)

**Comment by summeroff** [comment]
after you run "yarn  install" file should be in "node_modules\obs-studio-node"  subfolder.

**Comment by Tanoshi0625** [comment]
> after you run "yarn install" file should be in "node_modules\obs-studio-node" subfolder.

Yeah, it exists in subfolder, but terminal tell me Cannot find module.

**Comment by summeroff** [comment]
another issue might be .node file is technically a dll. and it might need some other dlls to load. like missing a "Visual C++ Redistributable" is popular issue. 
try to use https://www.dependencywalker.com to check if it has all dependency.

**Comment by aza547** [comment]
Check [this](https://stackoverflow.com/questions/66609529/electron-builder-installer-a-javascript-error-occurred-in-the-main-process-err/76105445#76105445) perhaps? I have an answer there that might help you. 
TLDR: Install this: https://aka.ms/vs/17/release/vc_redist.x64.exe/.
